src/components/Capture/capture.js

```diff
@@ -150,20 +150,20 @@ class Capture extends Component {
     if (isOfFileType(['pdf'], file)){
       // TODO: we still need to convert PDFs to base64 in order to send it to the websocket server
       // this code needs to be changed when HTTP protocol is implemented
-      const handlePDF = (base64) => this.handleFile(base64, file)
-      fileToBase64(file, handlePDF, this.onFileGeneralError)
-      return
+      pdfToBase64(file, (base64) => this.handleImage(base64, file), this.onFileGeneralError);
     }
 
     if (isOfFileType(['jpg','jpeg','png'], file)){
       //avoid rendering pdfs or other formats to image,
       //due to inconsistencies between different browsers and the back end
-      fileToLossyBase64Image(undefined, file,
-        lossyBase64 => this.handleFile(lossyBase64, file),
-        error => this.handleFile(undefined, file)
+      fileToLossyBase64Image(undefined, file, (lossyBase64) => {
+        fileToBlob(file, (blob) => this.handleImage(lossyBase64, blob),this.onFileGeneralError)
```

Comment:
I don't think we need to convert `File` to `Blob`, since `File` is inheriting from `Blob`. what do you think? Also, @stephencookdev do you have an opinion on this?